downloads/v1/versions/stable.json

```
@@ -392,7 +403,7 @@
             "displayName": "v38",
             "version": 38,
             "releaseDate": "2022-05-05",
-            "supported": false,
+            "supported": true,
```

@radnov the script keeps changing this to true
```suggestion
            "supported": false,
```